examples/1-hello-world/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "hello-world",
+  "version": "0.0.1",
+  "description": "Hello world example",
+  "main": "index.js",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/copilot-extensions"
+  },
```

I would remove all unnecessary keys, but add private: true

```suggestion
  "private": true,
```